fri/server/main.py

```diff
@@ -174,7 +188,9 @@ def getFilesList(dir):
 
 @app.route('/openJupyter/', methods=['POST'])
 def openJupyter():
-    cur_path = os.getcwd()
+    # cur_path = os.getcwd()
+    cur_path = os.path.dirname(os.path.abspath(__file__))
+    print(cur_path)

```

Comment:
I know you added this line
    print(cur_path)
for demo purposes. Please remove this line after today's demo.